templates/default.mustache

```diff
@@ -221,7 +221,7 @@ call <sid>hi("WarningMsg",    s:gui08, "", s:cterm08, "", "", "")
 call <sid>hi("WildMenu",      s:gui00, s:gui05, s:cterm00, s:cterm05, "", "")
 call <sid>hi("Title",         s:gui0D, "", s:cterm0D, "", "none", "")
 call <sid>hi("Conceal",       s:gui0D, s:gui00, s:cterm0D, s:cterm00, "", "")
-call <sid>hi("Cursor",        s:gui00, s:gui05, s:cterm00, s:cterm05, "inverse", "")
+call <sid>hi("Cursor",        s:gui05, s:gui00, "", "", "inverse", "")
```

Comment:
```suggestion
call <sid>hi("Cursor",        "", "", "", "", "reverse", "")
```
I'm not sure if you had a look at the branch I linked to previously, but I'm unsure as to why the change you've implemented should be used over this suggested change.